SafeguardSessions/README.md

```diff
@@ -13,40 +13,41 @@
 
 ## About the project
 
-The aim of One Identity Safeguard Power BI Connector (Connector) is to provide a solution for customers to visualize their audit data captured by [One Identity Safeguard for Priviledged Sessions] (SPS) in a highly configurable way compared to the on-box reporting system of SPS. The Connector uses basic authentication with local login method to connect to SPS and imports sessions metadata that matches the criteria given by the Connector's input parameters. After successful data retrieval, the Connector returns with two tables:
+The aim of One Identity Safeguard Power BI Connector (Power BI Connector) is to provide a solution for customers to visualize their audit data captured by [One Identity Safeguard for Priviledged Sessions] (SPS) in a highly configurable way compared to the on-box reporting system of SPS. The Power BI Connector uses basic authentication with the local login method to connect to SPS, and imports sessions metadata that matches the criteria specified in the Power BI Connector's input parameters. After successful data retrieval, the Power BI Connector returns two tables:
 
-- **Info**: contains information about the data fetch process for debugging purposes
+- **Info**: contains information about the data fetching process for debugging purposes
 - **Sessions**: contains the actual sessions metadata
 
-The project also a includes a Power BI report template to quickstart report creation and visualize your audit data.
+The project also a includes a Power BI Report Template (Report Template) to quickstart report creation and visualize your audit data.
```

Comment:
Power BI Connector and Report template aliases are not consistent.

It could have stay simply Connector and Report Template or it should have been Power BI Connector and Power BI Report template.

Reply:
Yeah, I know what you mean. This is because in the standalone guide, we refer to the connector as Power BI Connector and Report Template and the TW team asked to follow this convention here as well. Currently we cannot do anything about it, but noted it for revisit in the following release:

https://dev.azure.com/1id/SPS/_workitems/edit/416845

Reply:
Ack!